src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -68,23 +71,35 @@ const std::string kAppPolicyId = "fake-app-id";
 const uint32_t kDefaultTimeout = 1000u;
 const std::string kAppFolderName = "fake-app-name";
 const std::string kAppStorageFolder = "fake-storage";
-const std::string kSystemFilesPath = "/fake/system/files";
+const std::string kSystemFilesPath = "fake/system/files";
```

Comment:
@BSolonenko why do we need so deep folder structure? Please rename to "fake-system-files"